dubbo-config/dubbo-config-api/src/main/java/com/alibaba/dubbo/config/MonitorConfig.java

```diff
@@ -123,4 +125,12 @@ public void setDefault(Boolean isDefault) {
         this.isDefault = isDefault;
     }
 
+    public void setInterval(String interval){
+        this.interval = interval;
+    }
+
+    public String getInterval(){

```

Comment:
Who calls `getInterval()`?

Reply:
no one call `getInterval()` direct, this is used for fastjson serialize, so it can be passed to `URL`